test/splitter.js

```diff
@@ -3,26 +3,25 @@ const Splitter = artifacts.require("./Splitter.sol");
 
 contract('Splitter', async accounts => {
 
-    let gasCost = async receipt => {
-        const gasUsed = receipt.receipt.gasUsed;
-        const tx = await web3.eth.getTransaction(receipt.tx);
+    const { toBN, toWei } = web3.utils;
+
+    const getGasCost = async txObj => {
+        const gasUsed = txObj.receipt.gasUsed;
+        const tx = await web3.eth.getTransaction(txObj.tx);
         const gasPrice = tx.gasPrice;
 
-        return web3.utils.toBN(gasUsed * gasPrice);
+        return toBN(gasUsed * gasPrice);
```

Comment:
I would use `toBN(x).mul(toBN(y))` to make sure you don't ever use Javascript numbers.